src/Microsoft.Azure.ServiceBus/Filters/RuleDescription.cs

```diff
@@ -100,7 +111,7 @@ public string Name
 
             set
             {
-                if (string.IsNullOrWhiteSpace(value))
+                if (String.IsNullOrWhiteSpace(value))
                 {
```

Comment:
@SeanFeldman , that must have got changed automatically during resharper refactoring. Will change that back